app/src/androidTest/java/com/android/unio/ui/ScreenDisplayingTest.kt

```diff
@@ -120,6 +142,29 @@ class ScreenDisplayingTest {
     eventViewModel.loadEvents()
     eventViewModel.selectEvent(events.first().uid)
 
+    // Mocking the location permission checks
+    instrumentation = InstrumentationRegistry.getInstrumentation()
+    instrumentation.uiAutomation.executeShellCommand(
+        "pm grant ${InstrumentationRegistry.getInstrumentation().targetContext.packageName} android.permission.ACCESS_FINE_LOCATION")
+    instrumentation.uiAutomation.executeShellCommand(
+        "pm grant ${InstrumentationRegistry.getInstrumentation().targetContext.packageName} android.permission.ACCESS_COARSE_LOCATION")
```

Comment:
How does this work and why didn't you do the same in MapScreenTest ? I'm not sure to understand how these permission tests work.

Reply:
This is exactly the same as in the rule in the MapScreenTest. But you are right, I should remove this and also put the rule there.